src,permission: disable WASI when pm is enabled (#53124)
nodejs/node: doc/api/cli.md

Relative paths are NOT supported through the CLI flag.

### `--allow-wasi`

<!-- YAML
added: REPLACEME
-->

> Stability: 1.1 - Active development

> When using the [Permission Model][], the process will not be able to create any
> WASI instances by default.

Reviewer: ```suggestion
When using the [Permission Model][], the process will not be capable of creating any
WASI instances by default.
```
